functions/api/jisho.py

```diff
@@ -38,83 +42,103 @@ async def run_error(self, ctx, error):
         else:
             raise error
 
-    def _parse(self, keyword,  response=[]):
+    def _parse(self, keyword,  response = []):
         results = []
+        
         for data in response:
             readings: Set[str] = set()
             words: Set[str] = set()
             kanji_link: Set[str] = set()
-
+            
             for kanji in data['japanese']:
+                
                 reading: str = kanji.get('reading')
-                if reading and reading not in readings:
```

Comment:
This is intended, and should not be removed. If removed, code breaks. If you're intending on 'cleaning' this cog, you might probably want to, if possible, get the same result with a different approach.